src/Oceanostics.jl

```diff
@@ -38,6 +44,14 @@ validate_location(location, type, valid_location=(Center, Center, Center)) =
 validate_dissipative_closure(closure) = error("Cannot calculate dissipation rate for $closure")
 validate_dissipative_closure(::AbstractScalarDiffusivity{<:Any, ThreeDimensionalFormulation}) = nothing
 validate_dissipative_closure(closure_tuple::Tuple) = Tuple(validate_dissipative_closure(c) for c in closure_tuple)
+
+validate_buoyancy(buoyancy::Nothing) = throw(ArgumentError("Cannot calculate gravitational potential energy without a buoyancy model."))
+validate_buoyancy(buoyancy::Buoyancy{<:BuoyancyTracer, g}) where g =
```

Comment:
It would be trivial to implement potential energy with `BuoyancyTracer` though, right?

Reply:
I think you are right! Once the other comments here have been addressed could look at whether it is better to do in a new PR or make the change in this one?

Reply:
Agreed